Makefile

```diff
@@ -74,3 +87,7 @@ $(JSONNET_FMT): $(BIN_DIR)
 $(JSONNET_BUNDLER): $(BIN_DIR)
 	curl -L -o $(JSONNET_BUNDLER) "https://github.com/jsonnet-bundler/jsonnet-bundler/releases/download/v0.3.1/jb-$(shell go env GOOS)-$(shell go env GOARCH)"
 	chmod +x $(JSONNET_BUNDLER)
+
+$(KUBEVAL): $(BIN_DIR)
```

Comment:
fine with this for now, but maybe as a follow up we can use `bingo`? (I'm hooked)

Reply:
Yes, I was thinking the same, just didn't want to put even more into this PR :)